tests/unit/routers/test_user_router.py

```diff
@@ -1,38 +1,66 @@
 import pytest
 
-from app.schemas.user import UserSchema
+from app.schemas.user import RoleEnum, UserSchema
 
 
 @pytest.mark.usefixtures("client")
 class TestUserRouter:
-    """Base class for UserRouter tests."""
-
     def test_get_all_users(self):
-        """Test getting all users."""
-        response = self.client.get("/user/all")
+        response = self.client.get("/pegazzo/internal/user")
+        assert response.status_code == 200
+        data = response.json()
+        assert isinstance(data, list)
+        validated_users = [UserSchema.parse_obj(user) for user in data]

```

Comment:
it is a better approach just doing 
```suggestion
        assert all(UserSchema.validate(user) for user in data)
```